src/SessionManager.php

```diff
@@ -130,9 +130,13 @@ public function start($preserveStorage = false)
 
         session_start();
 
-        if ($oldSessionData instanceof \Traversable
-            || (! empty($oldSessionData) && is_array($oldSessionData))
-        ) {
+        if ($oldSessionData instanceof Storage\StorageInterface) {
```

Comment:
I think I don't understand something....
https://github.com/zendframework/zend-session/blob/7432b7dfde429a64e090c3cea7c8de99b1faeab5/src/Storage/StorageInterface.php#L23

Shouldn't we just have here: `if ($oldSessionData instanceof \Traversable) {` instead?

Reply:
AFAIK `toArray` is defined in `StorageInterface`.

Reply:
Yes, see my above comments https://github.com/zendframework/zend-session/pull/93#discussion_r153446047